### ISSUE TITLE
Malware detection, false positive?

### ISSUE THREAD

**Issue by DarthKaos92** [ItsCubeTime/Napoleon-Total-War-Intel-12th-gen-patch#1]
When I scanned the downloaded patched exe with Malwarebytes, it detected malware. I also got the same result when I patched my own napoleon exe. I presume this means it's a false positive but I figure it's worth mentioning. 

The malware in question is 'malware.heuristic.1003'

**Comment by ItsCubeTime** [comment]
We are after all modding an executable, so arguably its fair that an anti malware service would be suspicious on it.

This is one of the reasons that I decided to make a tutorial on how to patch the game yourself rather than only upload an executable, as I wouldn't expect everyone on the internet to trust somenoe uploading a modded executable like this.

But to answer the question
> Malware detection, false positive?

Yes, its a false positive. Ive been uploading software to Github for about 2 years now 💖The file uploaded here was created with the exact same steps as shown in the YT video linked in the readme file.

**Comment by DarthKaos92** [comment]
Yeah I suppose it makes sense anti malware would be suspicious of internal changes to an exe,  but it's nothing to worry about. Anyhow the game runs great now. Thank you for making napoleon playable again! 👍
